src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx

```diff
@@ -312,12 +320,32 @@ export default class OrganizationDiscover extends React.Component {
             {!data && <Intro updateQuery={this.updateFields} />}
             <EarlyAdopterMessage />
           </Flex>
-        </Flex>
+        </DiscoverBody>
       </Discover>
     );
   }
 }
 
+const DiscoverHeading = styled.h2`
+  font-size: 18px;
+  font-weight: normal;
+  color: ${p => p.theme.gray4};
+  margin: 0;
+`;
+
+const SidebarHeader = styled(Flex)`
+  border-bottom: 1px solid ${p => p.theme.borderLight};
+`;
+
+const DiscoverBody = styled(Flex)`
+  min-height: calc(100vh - 152px);
+`;
+
+const Sidebar = styled(Box)`
+  border-right: 1px solid ${p => p.theme.borderDark}; /* eventually make a react component to handle drag event */
```

Comment:
I don't think this comment belongs here, let's add it to the proposed future feature list